components/card.css

```diff
@@ -0,0 +1,126 @@
+
+/* Basic card */
+
+.card {
+    width: 310px;
+    border-radius: 10px;
+    min-height: 430px;
+    height: min-content;
+    display: flex;
+    flex-wrap: wrap;
+    padding-bottom: 10px;
+    overflow: hidden;
+    position: relative;
+  }
+  .card-head {
+    position: relative;
+    width: 100%;
+  }
+  .card-body {
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+  
+  /* Card image */
+  
+  .card-image {
+    border: none;
+    height: min-content;
+    width: 100%;
+    background-color: grey;
+  }
+  .card-image-img {
+      margin-left: auto;
+      margin-right: auto;
+    height: 100%;
+    display: block;
+  }
+  
+  /* card cross */
+  
+  .card-cross {
+      position: absolute;
```

Comment:
Can use helper class variables for clumping repeated properties like Position, top, right etc (under one class)